apps/etl/models.py

```diff
@@ -106,13 +107,13 @@ def __str__(self):
 
 class Transform(Resource):
     class Status(models.IntegerChoices):
-        SUCCESS = 1, "Success"
-        FAILED = 2, "Failed"
+        PENDING = 1, "Pending"
+        SUCCESS = 2, "Success"
+        FAILED = 3, "Failed"
 
-    extraction = models.ForeignKey(
-        ExtractionData, on_delete=models.PROTECT, null=True, blank=True, verbose_name=_("extraction")
-    )
+    extraction = models.ForeignKey(ExtractionData, on_delete=models.PROTECT, verbose_name=_("extraction"))
     status = models.IntegerField(verbose_name=_("transform status"), choices=Status.choices)
+    is_loaded = models.BooleanField(default=False, help_text="Check if transformation is completed. It is helpful in debugging")
```

Comment:
```suggestion
    is_loaded = models.BooleanField(default=False, help_text="Track whether transformer data has been successfully loaded into the PyStacLoadData table. This flag can be used to re-populate the data in case of any issues with the transformer.")
```